tools/Azure.Mcp.Tools.EventHubs/src/Models/EventHubsNamespaceDetails.cs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Text.Json.Serialization;
+
+namespace Azure.Mcp.Tools.EventHubs.Models;
+
+public record EventHubsNamespaceDetails(
+    [property: JsonPropertyName("name")] string Name,
```

Comment:
Is this an updated way to specify Models?  AFAIK, it's implied that the name of the JSON property is a camel cased version of the property's name.  I usually see the [JsonPropertyName] attribute when the JSON object name is something different.

Reply:
In this PR as well, it is inconsistent.  Should stick to one.

Reply:
I'll update it to follow the pattern in NamespaceGetOptions